src/core_plugins/kibana/common/field_formats/field_formats.js

```diff
@@ -0,0 +1,45 @@
+export class FieldFormatsService {
+  constructor() {
+    this.fieldFormats = new Map();
```

Comment:
As far as I can tell this is intended for internal use only, it should be prefixed with an `_`.